libslide/src/evaluator_rules/registry.rs

```diff
@@ -0,0 +1,90 @@
+mod fn_rules;
+
+use super::rule::Rule;
+use super::unbuilt_rule::UR;
+
+use fn_rules::*;
+
+// TODO: will allow lint in a future iteration when we support removing rules.
+#[allow(dead_code)]
+pub enum RuleName {
+    Add = 0,
+    Subtract,
+    Multiply,
+    Divide,
+    Modulo,
+    Exponentiate,
+    Posate,
+    Negate,
+    AdditiveIdentity,
+    ReorderConstants,
+    UnwrapParens,
+    UnwrapBraces,
+}
+
+static DEFAULT_RULESET: &[UR] = &[
+    UR::F(add),                           // Add
+    UR::F(subtract),                      // Subtract
+    UR::F(multiply),                      // Multiply
+    UR::F(divide),                        // Divide
+    UR::F(modulo),                        // Modulo
+    UR::F(exponentiate),                  // Exponentiate
+    UR::F(posate),                        // Posate
+    UR::F(negate),                        // Negate
+    UR::S("_a + 0 -> _a"),                // AdditiveIdentity
+    UR::M(&["#a + $b -> $b + #a"]),       // ReorderConstants
+    UR::M(&["($a) -> $a", "(#a) -> #a"]), // UnwrapParens
+    UR::M(&["[$a] -> $a", "[#a] -> #a"]), // UnwrapBraces
+];
```

Comment:
UR is not a great name. Lets use unbuilt_rule or something.

Reply:
Would you be comfortable with changing to `UnbuiltRule` but importing it as `UR`? I just wanted to do it for succinctness so we could fit all this in the array.

Reply:
I think your update is fine